submissions/Jemikalajah-Philip/php/create.php

```diff
@@ -0,0 +1,57 @@
+<?php
+if (isset($_POST['submit'])){
+    require "config.php";
+    require "helper.php";
+    
+    try {
+        $connection = new PDO ($dsn, $username, $password, $options);
+        $newProduct = array(
+            "itemname" => $_POST['itemname'],
+            "quantity" => $_POST['quantity'],
+            "manufacturer" => $_POST['manufacturer'],
+            "about" => $_POST['about'],
+            "brand_id" => $_POST['brands'],
+        );
+        $sql = sprintf(
+            "INSERT INTO %s (%s) values (%s)",
+            "products",
+            implode(", ", array_keys($newProduct)),
+            ":".implode(", :", array_keys($newProduct))
+        );
+        $statement = $connection->prepare($sql);
+        $statement->execute($newProduct);
+    }catch(PDOException $error){
+        echo $sql. "<br>" . $error->getMessage();
+    }
+}
+?>
+
+<?php require "header.php"; require "fetch.php";  ?>
+
+<?php
+if (isset($_POST['submit']) && $statement){ ?>
+> <?php echo $_POST['itemname']; ?> successfully added.
+<?php } ?>
+
+    <h2>Add a Product</h2>
+    <div class="signin">
+    <form  method="post">
+        <label class="add_product" for="itemname">Name of Product</label>
+        <input type="text" name="itemname" id="itemname">
+        <?php session_start(); ?>
+        <select name="brands">
+            <?php for($i=0; $i < count($_SESSION['mapped']); $i++) { ?>
+                <option value= <?php echo $i;?>> <?php echo $_SESSION["mapped"][$i] ?></option>
```

Comment:
Since your brands are mapped, you should be getting the keys and values here. Try using a foreach statement